Sources/PointFree/Mailgun/SendEmail.swift

```diff
@@ -0,0 +1,56 @@
+import Either
+import Html
+import Prelude
+
+func sendEmail(
+  from: String,
+  to: [String],
+  subject: String,
+  content: Either3<String, [Node], (String, [Node])>,
```

Comment:
this `sendEmail` is just like a nicer interface to the Mailgun one that exposes everything their API has to offer.

a fun / interesting / weird thing we are doing here is using our new `Either3` type! it allows us to support the user sending us plain text, html (from which we will derive plain text), or both! Haskell has a `These` type for this, but honestly the more generic `Either3` seems better!